src/components/structures/ScrollPanel.tsx

```diff
@@ -937,7 +937,7 @@ export default class ScrollPanel extends React.Component<IProps> {
             >
                 { this.props.fixedChildren }
                 <div className="mx_RoomView_messageListWrapper">
-                    <ol ref={this.itemlist} className="mx_RoomView_MessageList" aria-live="polite">
+                    <ol ref={this.itemlist} className="mx_RoomView_MessageList" aria-live="polite" data-cy="message-list">
```

Comment:
This isn't great naming, we use ScrollPanel even in RoomDirectory, I know the classes are already poorly named but lets please not propagate it

Reply:
Any suggestions?

It's pretty ambiguous as `data-cy="scroll-panel"` and probably shouldn't live here in that case.

We could passthrough `timelineRenderingType` from `TimelinePanel` -> `MessagePanel` -> `ScrollPanel` to try to get a more accurate name. Or probably better to just do a generic pass-through of `data-` attributes on those elements to add them on whatever DOM container. This way we could just do the following and it work work:

```jsx
<TimelinePanel data-cy={`timeline-${this.context.timelineRenderingType}`}>
```